modules/apps/headless/headless-delivery/headless-delivery-impl/rest-openapi.yaml

```diff
@@ -2543,6 +2561,24 @@ components:
                     readOnly: true
                     type: object
             type: object
+        ViewportColumnConfigurationDefinition:
+            # @review
+            properties:
+                size:
+                    maximum: 12
+                    minimum: 1
+                    type: integer
+            type: object
+        ViewportRowConfigurationDefinition:
+            # @review
+            properties:
+                modulesPerRow:
+                    type: integer
+                reverseOrder:
+                    type: boolean
+                verticalAlignment:
+                    type: string
```

Comment:
We can use an enum so it's easier for users to know which are the supported values